src/apple/macos/ffi.rs

```diff
@@ -14,6 +14,57 @@ use libc::{c_int, mach_port_t, size_t};
 
 pub(crate) use crate::sys::ffi::*;
 
+pub const MAXPATHLEN: usize = libc::PATH_MAX as usize;
+
+#[repr(C)]
+#[derive(Copy, Clone)]
+pub struct vinfo_stat {
```

Comment:
About the constants/types you added, I'll try to add them into `libc` to not have to support this in `sysinfo` directly (and also because their checks are much better than the almost non-existent ones in `sysinfo`).